pydle/connection.py

```diff
@@ -127,5 +127,5 @@ def stop(self):
         self.writer.write(data)
         await self.writer.drain()
 
-    async def recv(self):
-        return await self.reader.readline()
+    async def recv(self, *, timeout=None):
```

Comment:
If we are going to make `timeout` a keyword only argument by swallowing remaining `*args`, we should raise a warning of any args are present as their presence indicates a usage error.

Reply:
That is exactly what the lone `*` does:

```python
>>> def foo(*, kwonly=42):
...     pass
...
>>> foo(4)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: foo() takes 0 positional arguments but 1 was given
```

Reply:
TIL. I wasn't aware of that language feature.